src/test_utils/katana/genesis.rs

```diff
@@ -326,7 +300,7 @@ impl KatanaGenesisBuilder<Initialized> {
             number: 0,
             timestamp: 0,
             sequencer_address: self.compute_starknet_address(self.coinbase)?,
-            gas_prices: GasPrices::default(),
+            gas_prices: GasPrices { eth: 1, strk: 0 },
```

Comment:
I think Elias also commented somewhere about not needed this as gas, could we try using `GasPrices::default()`? I'm not sure why I ever used `eth: 1`.

Reply:
fixed